src/style.css

```diff
@@ -1,3 +1,56 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --linkcolor: #00acdc;
+}
+
+.logo {
+  display: flex;
```

Comment:
Do we really need this style? `.logo` has no children, so using flex is not needed.